test/services/school/index.test.js

```diff
@@ -34,6 +34,18 @@ describe('school service', () => {
 			delete sampleSchoolData._id;
 		});
 
+		it('load the school results with pagination', async () => {
+			const result = await schoolService.find();
+			expect(result).to.not.be.empty;
+		});
+
+		it('load the school results without pagination', async () => {
+			const result = await schoolService.find({
+				paginate: false,
+			});
+			expect(result).to.not.be.empty;
+		});
+
```

Comment:
These tests are not really helpful, because pagination is feathers base functionality. What ou could test would be if the custom years feature (addYearsToSchool method from above) works in these two cases.

Reply:
Test Suggestion:
```
const compareSchoolYears = (schoolYears, defaultYears) => {
		schoolYears.forEach((resultYear, idx) => {
			const defaultYear = defaultYears[idx];
			expect(resultYear.name).to.be.equal(defaultYear.name);
		});
	};

it('load the school results with pagination', async () => {
			const result = await schoolService.find();
			result.data.forEach((school) => compareSchoolYears(school.years.schoolYears, defaultYears));
		});

it('load the school results without pagination', async () => {
	const result = await schoolService.find({
			paginate: false,
	});
	result.forEach((school) => compareSchoolYears(school.years.schoolYears, defaultYears));
});

```